src/main/java/be/syntra/devshop/DevshopBack/exceptions/NotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package be.syntra.devshop.DevshopBack.exceptions;
+
+import org.springframework.http.HttpStatus;
+
+public class NotFoundException extends CustomException {
```

Comment:
if you're creating a non-specific exception, I advise to delete this class and use the CustomException directly

Reply:
fixed